ui/components/multichain/account-list-menu/account-list-menu.js

```diff
@@ -214,23 +208,19 @@ export const AccountListMenu = ({
             />
           </Box>
         ) : null}
-        {
-          ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
-          actionMode === ACTION_MODES.ADD_BITCOIN ? (
-            <Box paddingLeft={4} paddingRight={4} paddingBottom={4}>
-              <CreateBtcAccount
-                onActionComplete={(confirmed) => {
-                  if (confirmed) {
-                    onClose();
-                  } else {
-                    setActionMode(ACTION_MODES.LIST);
-                  }
-                }}
-              />
-            </Box>
-          ) : null
-          ///: END:ONLY_INCLUDE_IF
-        }
+        {actionMode === ACTION_MODES.ADD_BITCOIN ? (
```

Comment:
Even though were not supposed to have this action mode without the new menu entry, maybe we could still check for `bitcoinSupportEnabled` just in case?

```suggestion
        {bitcoinSupportEnabled && actionMode === ACTION_MODES.ADD_BITCOIN ? (
```